packages/mgt-components/src/components/mgt-person-card/mgt-person-card.scss

```diff
@@ -123,10 +138,60 @@ $person-card-details-item-font-size: var(--person-card-details-item-font-size, 1
   }
 
   .section-nav {
-    display: flex;
-    padding: 0 14px;
     border-bottom: 1px solid set-var(tab__line__color, $theme-default, $common);
+    height: 36px;
+    padding-bottom: 7px;
+    fluent-tabs {
+      background-color: $background-color;
+      grid-template-columns: minmax(1px, 0.1fr);
+      .activeIndicator {
+        width: 40px !important;
+      }
+      .tablist {
+        padding-left: 20px !important;
+      }
+      fluent-tab-panel {
+        max-height: 360px;
+        min-height: 360px;
+        overflow-y: auto;
+        overflow-x: hidden;
+        padding: 0px;
+        border-top: 1px solid set-var(tab__line__color, $theme-default, $common);
+        .inserted {
+          max-width: var(--mgt-flyout-set-width, 375px);
+          min-width: var(--mgt-flyout-set-width, 360px);
+        }
+        .overviewPanel {
+          max-width: var(--mgt-flyout-set-width, 375px);
+        }
 
+        &::-webkit-scrollbar {
```

Comment:
Does the default scrollbar not fit our design? This won't work in FireFox.

Reply:
The default scrollbar doesn't seem to fit our designs/fluent designs... Is it ok to fallback to a regular scrollbar? 

I'll discuss with Ying again:
@yyiinnggzz

Reply:
Just to clean this up:

We decided to use a fallback style, so its not exact but fairly close with the css:
```css
scrollbar-width: thin;
```